syft/workers/base.py

```diff
@@ -348,6 +358,8 @@ def execute_command(self, message: tuple) -> PointerTensor:
         command_name = command_name
         # Handle methods
         if _self is not None:
+            if type(_self) == int:
```

Comment:
Is this really needed? If so we need to find a better way to deal with it.

Reply:
This is needed to provide an id of an object, whose method will be called. This is the easiest way to reference an object at the location. And the only thing known by the pointer.